modules/publication/jsx/uploadForm.js

```diff
@@ -195,14 +191,18 @@ class PublicationUploadForm extends React.Component {
     let formClass = 'col-md-12 col-lg-12';
     if (!this.props.editMode) {
       createElements = [
-        <h3 className="col-md-offset-3 col-lg-offset-3">Propose a new project</h3>,
-        <TextboxElement
-          name="title"
-          label="Title"
-          onUserInput={this.setFormData}
-          required={true}
-          value={this.state.formData.title}
-        />,
+        <div key={'propose_new_project'}>
+          <h3 className="col-md-offset-3 col-lg-offset-3">
+            Propose a new project
+          </h3>,
+          <TextboxElement
+            name="title"
+            label="Title"
+            onUserInput={this.setFormData}
+            required={true}
+            value={this.state.formData.title}
+          />,
+        </div>,
```

Comment:
createElements is an array of jsx. i don't think you need the div that you added here and it's also incorrect syntax right? you're nesting elements of an array within div tags.

Reply:
@zaliqarosli The extra div is fixing the error "Warning: Each child in an array or iterator should have a unique "key" prop." from displaying in the console. I'll change the formatting to be createElement = () instead of an array. Currently we don't have a way to supply a key to TextboxElement or I would find the extra div unnecessary